packages/odyssey-storybook/src/components/odyssey-mui/Card/Card.stories.tsx

```diff
@@ -174,7 +182,20 @@ export const ClickableWithoutImage: StoryObj<CardProps> = {
 export const ButtonWithoutImage: StoryObj<typeof Card> = {
   render: ({ ...props }) => (
     <Box sx={{ maxWidth: 262 }}>
-      <Card {...props} />
+      <Card
+        overline={props.overline}
+        title={props.title}
+        description={props.description}
+        image={props.image}
+        menuButtonChildren={
+          <>
+            <MenuItem>Menu option</MenuItem>
+            <MenuItem>Menu option</MenuItem>
+            <MenuItem>Menu option</MenuItem>
+          </>
+        }
+        button={<Button variant="primary" label="Button" />}
+      />
```

Comment:
Just for posterity, I'd number these:
```suggestion
        menuButtonChildren={
          <>
            <MenuItem>Menu option 1</MenuItem>
            <MenuItem>Menu option 2</MenuItem>
            <MenuItem>Menu option 3</MenuItem>
          </>
        }
        button={<Button variant="primary" label="Button" />}
      />
```